feature/src/main/java/com/gdg/feature/calendar/CalendarRoute.kt

```diff
@@ -235,7 +235,7 @@ fun CalendarInfoBox(data: ScheduleEntity) {
         ) {
             Text(
                 modifier = Modifier.padding(end = 8.dp),
-                text = data.location,
+                text = data.location.chunked(45).joinToString("\n"), // 긴 문구 단어 단위로 줄바꿈
```

Comment:
P1: 45자 단위로 제한할 경우 기기 크기에 따라 줄바꿈이 안예쁘게 이뤄지기 때문에 글자수를 기준으로 줄바꿈해서는 안됩니다. 줄바꿈을 공백으로 변경하는게 훨씬 깔끔해집니답. 그리고 해당 text modifier에 weight(1f) 적용해주세요
```suggestion
                text = data.location.replace("\n", " ")
```